src/main/java/io/ncbpfluffybear/supserv/items/FarmersHoe.java

```diff
@@ -0,0 +1,61 @@
+package io.ncbpfluffybear.supserv.items;
+
+import io.github.thebusybiscuit.slimefun4.core.handlers.ItemUseHandler;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import io.github.thebusybiscuit.slimefun4.implementation.items.SimpleSlimefunItem;
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import me.mrCookieSlime.Slimefun.cscorelib2.protection.ProtectableAction;
+import org.bukkit.CropState;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.block.data.Ageable;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.Nonnull;
+import java.util.Arrays;
+import java.util.List;
+
+public class FarmersHoe  extends SimpleSlimefunItem<ItemUseHandler> {
+
+    public FarmersHoe(Category category, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe, ItemStack recipeOutput) {
+        super(category, item, recipeType, recipe, recipeOutput);
+
+    }
+
+    List<Material> farmables = Arrays.asList(Material.DIRT, Material.GRASS_BLOCK, Material.COARSE_DIRT);
+    @Nonnull
+    @Override
+    public ItemUseHandler getItemHandler() {
+        return e -> {
+
+            Player p = e.getPlayer();
+            ItemStack item = e.getItem();
+
+            if (!isItem(item))
+                return;
+
+            if(!e.getClickedBlock().isPresent()) return;    // Check if clicked block is null > return
```

Comment:
Use open and close braces for all statements, I'd like it if my code was formatted following [Google's Style Guide](https://google.github.io/styleguide/javaguide.html) (This is also standard for Slimefun PRs so I just adopted their practices, also makes code look cleaner if everything is uniform)